src/app/modules/group/pages/group-user/group-user.component.ts

```diff
@@ -1,24 +1,80 @@
-import { Component } from '@angular/core';
+import { Component, OnDestroy, OnInit, ViewChild } from '@angular/core';
 import { User, GetUserService } from '../../http-services/get-user.service';
 import { mapToFetchState } from '../../../../shared/operators/state';
-import { Observable } from 'rxjs';
-import { ActivatedRoute } from '@angular/router';
-import { switchMap } from 'rxjs/operators';
+import { Observable, Subscription } from 'rxjs';
+import { ActivatedRoute, NavigationEnd, Router, RouterLinkActive } from '@angular/router';
+import { delay, switchMap, map, filter, startWith } from 'rxjs/operators';
+import { contentInfo } from '../../../../shared/models/content/content-info';
+import { CurrentContentService } from '../../../../shared/services/current-content.service';
+import { UserSessionService } from '../../../../shared/services/user-session.service';
 
 @Component({
   selector: 'alg-group-user',
   templateUrl: './group-user.component.html',
   styleUrls: [ './group-user.component.scss' ]
 })
-export class GroupUserComponent {
+export class GroupUserComponent implements OnInit, OnDestroy {
+  @ViewChild('progress') progress?: RouterLinkActive;
+  @ViewChild('personalData') personalData?: RouterLinkActive;
+
   readonly state$ = this.route.params.pipe(
     switchMap(({ id }) => this.getUser$(id)),
-    mapToFetchState()
+    mapToFetchState(),
+  );
+
+  readonly currentUserGroupId$ = this.userSessionService.userProfile$.pipe(
+    delay(0),
+    map(userProfile => userProfile.groupId),
   );
 
-  constructor(private route: ActivatedRoute, private getUserService: GetUserService) { }
+  private subscription?: Subscription;
+
+  isInitPagePersonal = false;
+
+  constructor(
+    private route: ActivatedRoute,
+    private router: Router,
+    private getUserService: GetUserService,
+    private userSessionService: UserSessionService,
+    private currentContent: CurrentContentService,
+  ) {}
+
+  ngOnInit(): void {
+    this.subscription = this.router.events
+      .pipe(
+        filter(event => event instanceof NavigationEnd),
+        startWith(null),
+      )
+      .subscribe(() => {
+        this.updateBreadcrumbs();
+      });
```

Comment:
Just follow your state and display "Users \ `user|caption` \ Personal info or Progress" when you know your user and  "Users \  ... \ Personal info or Progress" on loading/error.
Like in group-by-id.component.ts:50-59, just map your state to a breadcrumb, and then set it in your subscribe.

Reply:
Fixed